yt/visualization/profile_plotter.py

```diff
@@ -416,11 +417,7 @@ def _initialize_instance(cls, obj, profiles, labels, plot_specs, y_log):
         obj._font_color = None
         obj.profiles = ensure_list(profiles)
         obj.x_log = None
-        obj.y_log = {}
-        if y_log is not None:
-            for field, log in y_log.items():
-                field, = obj.profiles[0].data_source._determine_fields([field])
-                obj.y_log[field] = log
+        obj.y_log = sanitize_field_tuple_keys(y_log, obj.profiles[0].data_source) or {}
```

Comment:
since you moved the logic for fields = y_fields above this block, maybe adding this logic for the y_log immediately after would be good too. it would make this profile creation cleaner.

Reply:
I am not sure what you mean here. Aren't those very distant blocks from 2 different method def ? (even if the name `_initialize_instance` puzzles me...)

Reply:
Ha, yes sorry. Ignore this comment. I clearly hadn't had enough ☕ yet.